### ISSUE TITLE
Error on Deno runtime

### ISSUE THREAD

**Issue by irustm** [paulmillr/noble-secp256k1#30]
<img width="1123" alt="imp" src="https://user-images.githubusercontent.com/16316579/140982185-ffe074ec-db88-42ba-9f80-3100abc44a6f.png">

This error after 1.2.10 version

```
error: TS2580 [ERROR]: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
  const nodeRequire = typeof module !== 'undefined' && typeof require === 'function';
                             ~~~~~~
    at https://deno.land/x/secp256k1@1.3.0/index.ts:1139:30

TS2580 [ERROR]: Cannot find name 'require'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
  const nodeRequire = typeof module !== 'undefined' && typeof require === 'function';
                                                              ~~~~~~~
    at https://deno.land/x/secp256k1@1.3.0/index.ts:1139:63

TS2580 [ERROR]: Cannot find name 'require'. Do you need to install type definitions for
```

https://github.com/paulmillr/noble-secp256k1/compare/1.2.9...1.2.10#diff-e727e4bdf3657fd1d798edcd6b099d6e092f8573cba266154583a746bba0f346R858-R860

**Comment by paulmillr** [comment]
Use 1.3.0

**Comment by irustm** [comment]
In 1.3.0 same error

```
error: TS2580 [ERROR]: Cannot find name 'module'. Do you need to install type definitions for node? Try `npm i --save-dev @types/node`.
  const nodeRequire = typeof module !== 'undefined' && typeof require === 'function';
                             ~~~~~~
    at https://deno.land/x/secp256k1@1.3.0/index.ts:1139:30
```

**Comment by paulmillr** [comment]
Fixed in 1.3.2. Make sure to read README for updated usage with Deno.
